presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -324,6 +324,7 @@ public final class SystemSessionProperties
     public static final String OPTIMIZER_USE_HISTOGRAMS = "optimizer_use_histograms";
     public static final String WARN_ON_COMMON_NAN_PATTERNS = "warn_on_common_nan_patterns";
     public static final String INLINE_PROJECTIONS_ON_VALUES = "inline_projections_on_values";
+    public static final String SINGLE_WORKER_EXECUTION_ENABLED = "single_worker_execution_enabled";
```

Comment:
Let's call it `single_node_execution_enabled`. When running on Java and `schedule-on-coordinator` is enabled the query may run completely on coordinator.